app/controllers/users_controller.rb

```diff
@@ -0,0 +1,2 @@
+class UsersController < ApplicationController
+end
```

Comment:
- [ ] UsersController is required to have `index` and `show` actions for this milestone. You may not need to define it but declare it. Kindly add them. :pray: